backend/handlers/handlers.go

```diff
@@ -181,9 +189,13 @@ func NewHandlers(
 	getAPIRouter(apiRouter)("/coins/btc/headers/status", handlers.getHeadersStatus("btc")).Methods("GET")
 	getAPIRouter(apiRouter)("/certs/download", handlers.postCertsDownloadHandler).Methods("POST")
 	getAPIRouter(apiRouter)("/certs/check", handlers.postCertsCheckHandler).Methods("POST")
+	getAPIRouter(apiRouter)("/connectbase", handlers.postConnectBaseHandler).Methods("POST")
```

Comment:
but connect and disconnect also unuder the bitboxBaseRouter, so it's `/bitboxbases/connect`?